packages/data/domain/validation.schemas.ts

```diff
@@ -20,10 +20,30 @@ export type NewApplicationModel = z.infer<typeof newApplicationSchema>;
 export const interviewSelectSchema = createSelectSchema(interviews);
 export type InterviewModel = z.infer<typeof interviewSelectSchema>;
 
+export const interviewWithCommentSchema = interviewSelectSchema.extend({comments: z.array(z.object({id: z.string(), interview_id: z.string(), comment_date: z.number(), comment: z.string(), pinned: z.boolean()}))})
+export type InterviewWithCommentModel = z.infer<typeof interviewWithCommentSchema>
+
+
 export const newInterviewSchema = createInsertSchema(interviews).omit({
   id: true,
 });
 export type NewInterviewModel = z.infer<typeof newInterviewSchema>;
 
 export const interviewUpdateSchema = createUpdateSchema(interviews);
-export type UpdateInterviewModel = z.infer<typeof interviewUpdateSchema>;
\ No newline at end of file
+export type UpdateInterviewModel = z.infer<typeof interviewUpdateSchema>;
+
+
+export const interviewCommentSchema = createSelectSchema(interviewComments)
+export type InterviewCommentModel = z.infer<typeof interviewCommentSchema>
+
+export const newIntervewCommentSchema = createInsertSchema(interviewComments).omit({
+  id: true,
+  interview_id:true
+})
+
+export type NewInterviewCommentModel = z.infer<typeof newIntervewCommentSchema> 
+
+
+
+
+
```

Comment:
I think this would be a good time to introduce a code formatting tool... I'll address this next.